mono/metadata/object.c

```diff
@@ -538,7 +546,7 @@ mono_runtime_class_init_full (MonoVTable *vtable, MonoError *error)
 	mono_type_initialization_unlock ();
 
 	if (do_initialization) {
```

Comment:
I'm erring toward less churn here and still maybe too much.
I just want to be able to place coop handle scopes, and was erring toward one large one.
I didn't really read the code and decide it "needed" cleanup otherwise.